### DISCUSSION TITLE
无法验证流量包

### DISCUSSION THREAD

**Discussion by rockmelodies** [jasonish/docker-suricata#33] [General]
<img width="1076" alt="image" src="https://user-images.githubusercontent.com/24653177/218997891-6ae30c53-7a25-4f59-8d55-827df9e15021.png">

**Comment by jasonish** [comment]
Try specifying the full path to the PCAP to where it will be found within the container, or use `-w /tmp` to set the working directory to `/tmp/ where it looks like the pcaps might be found.
